content/en/blog/2023/humans-of-otel.md

```diff
@@ -0,0 +1,458 @@
+not forget how we had not one, but TWO Observability Days in 2023 - one for
+[KubeCon Europe in Amsterdam](https://shorturl.at/osHRX), and one for
+[KubeCon North America in Chicago](https://shorturl.at/kAEQX).
+
+None of this would've been possible without the awesome humans behind
+OpenTelemetry. Whether you're a maintainer, contributor, or practitioner, we
+want y'all to know that what you do matters!
+
+And so, to wrap up 2023, I interviewed some of the folks who have been involved
+with OTel, both past and present:
+
+- [Tyler Yahn](https://www.linkedin.com/in/tyler-yahn/)
+- [Amy Tobey](https://www.linkedin.com/in/amytobey/)
+- [Ted Young](https://www.linkedin.com/in/ted-young/)
+- [Carter Socha](https://www.linkedin.com/in/cartersocha/)
+- [Bogdan Drutu](https://www.linkedin.com/in/bogdan-drutu-7909602a/)
+- [Constance Caramanolis](https://www.linkedin.com/in/ccaramanolis/)
+- [Juraci Paixāo Kröhling](https://www.linkedin.com/in/jpkroehling/)
+- [Jacob Aronoff](https://www.linkedin.com/in/jaronoff97/)
+- [Alex Boten](https://www.linkedin.com/in/codeboten/)
+- [Purvi Kanal](https://www.linkedin.com/in/🚀-purvi-kanal-4a36089a/)
+
+Special thanks to [Reese Lee](https://www.linkedin.com/in/reese-lee/) for the
+camera work!
+
+You can watch the full recording here:
+
+{{<youtube coPrhP_7lVU>}}
+
+<br/>Thanks to everyone who has contributed to OpenTelemetry to date, and we
+look forward to your contributions in 2024! 🎉
+
+## Transcript
+
+If reading is more your jam, check out the transcript of our conversations
+below.
+
+### 1- Meet the Humans of OTel
+
+**TYLER YAHN:** I'm Tyler Yahn. I am a maintainer for the OpenTelemetry Go SIG.
+We're working on some auto-instrumentation there, and specification.
+
+**AMY TOBEY:** I'm Amy Tobey. I am senior principal engineer for digital
+interconnection at Equinix. I maintain a tool called OTel CLI.
+
+**ADRIANA VILLELA:** Oh, you maintain the OTel CLI!
+
+**AMY TOBEY:** Yeah, that's my project. Yeah. So that mostly just has Traces
+right now. And I've been meaning to implement Logs and Metrics for a while and I
+think like Logs just went GA recently, so it's time to do it. But Traces have
+been so effective and people really like it that I haven't really had a lot of
+demand for them.
+
+**ADRIANA VILLELA:** Is the OTel CLI part of OpenTelemetry?
+
+**AMY TOBEY:** It's not yet. I maintain it on our Equinix Labs GitHub account.
+It's not a lot of process. Mostly it's just me with a few folks like Alex and
+others that throw me a PR every now and then. But I've thought about bringing it
+back to the community. But I'd have to maybe be not as so far away from the
+standards as I am right now because doing it in the command line, a lot of the
+standards don't really translate very well. So I've strayed a little bit away
+from the standards in a few places. It would make sense. And I've talked to
+Austin about it.
+
+**ADRIANA VILLELA:** Hey, I've got Ted Young with me! Hello!
```

Comment:
```suggestion
**ADRIANA VILLELA:** Hey, I've got Ted Young with me! Hello!

**TED YOUNG:** Hello!
```

I think I heard Ted say Hello, so it looks better instead of having Carter speak next